x-pack/plugins/security_solution/public/flyout/right/content.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlyoutBody } from '@elastic/eui';
+import type { VFC } from 'react';
```

Comment:
We'll have to update this whenever we move to React 18 since it's deprecated in the most recent version